tests/src/test/java/alluxio/client/cli/fs/command/PersistCommandTest.java

```diff
@@ -259,16 +263,42 @@ public void persistParallelFilesGtParallelism() throws Exception {
     persistParallel(2, 1);
   }
 
-  public void persistParallel(int totalFiles, int parallelism) throws Exception {
-    int fileSize = 30;
+  @Test
+  public void persistShortTimeout() throws Exception {
+    createUnpersistedFiles("testPersistTimeout", 100, 50);
```

Comment:
strange to see an absolute path that doesn't start with "/" - I'd put the "/" here for clarity instead of in `createUnpersistedFiles`

Reply:
👍